<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 44, 52);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:0px;
background-color: rgb(68, 158, 241);
color: rgb(255, 255, 255);
font-weight: bold
}

QPushButton::pressed {
background-color: rgb(93, 127, 247);
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="databaseTab">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>1080</width>
      <height>750</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="sideColor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>30</width>
      <height>730</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(68, 158, 241);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOpen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:0px;
background-color: rgb(34, 35, 40);
color: rgb(255, 255, 255);
font-weight: bold
}

QPushButton::pressed {
border-right: 5px;
border-right-color:red;
}</string>
    </property>
    <property name="text">
     <string>Open database</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="buttonColor_1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>30</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(68, 158, 241);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttonColor_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>30</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(68, 158, 241);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttonColor_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>30</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(68, 158, 241);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCreate">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:0px;
background-color: rgb(34, 35, 40);
color: rgb(255, 255, 255);
font-weight: bold
}

QPushButton::pressed {
border-right: 5px;
border-right-color:red;
}</string>
    </property>
    <property name="text">
     <string>Create database</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSettings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border:0px;
background-color: rgb(34, 35, 40);
color: rgb(255, 255, 255);
font-weight: bold
}

QPushButton::pressed {
border-right: 5px;
border-right-color:red;
}</string>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>buttonSettings</zorder>
   <zorder>buttonCreate</zorder>
   <zorder>pushButton</zorder>
   <zorder>databaseTab</zorder>
   <zorder>sideColor</zorder>
   <zorder>buttonOpen</zorder>
   <zorder>buttonColor_1</zorder>
   <zorder>buttonColor_2</zorder>
   <zorder>buttonColor_3</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menummm">
    <property name="title">
     <string>mmm</string>
    </property>
   </widget>
   <addaction name="menummm"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOpen</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonColor_1</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCreate</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonColor_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSettings</sender>
   <signal>toggled(bool)</signal>
   <receiver>buttonColor_3</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
